### ISSUE TITLE
FormioResourceService.formFormio has no "userPermission"

### ISSUE THREAD

**Issue by dmenne** [formio/angular-formio#368]
` "angular-formio": "^4.0.2"`
Displaying a `FormioResourceViewComponent`

Seems to work ok, but gives an error message:

```
Error: Uncaught (in promise): TypeError: _this.service.formFormio.userPermissions is not a function
TypeError: _this.service.formFormio.userPermissions is not a function
```
(incorrectly spelled *userPermission* in above title)

```
import { Component } from '@angular/core';
import { ActivatedRoute } from '@angular/router';
import { FormioAuthService } from '../auth/auth.service';
import { FormioResourceService } from './resource.service';
var FormioResourceComponent = /** @class */ (function () {
    function FormioResourceComponent(service, route, auth) {
        this.service = service;
        this.route = route;
        this.auth = auth;
        this.perms = { delete: false, edit: false };
    }
    /**
     * @return {?}
     */
    FormioResourceComponent.prototype.ngOnInit = /**
     * @return {?}
     */
    function () {
        var _this = this;
        this.service.loadResource(this.route);
        this.service.formLoaded.then((/**
         * @param {?} form
         * @return {?}
         */
        function (form) {
            _this.auth.ready.then((/**
             * @return {?}
             */
            function () {
                _this.service.resourceLoaded.then((/**
                 * @param {?} resource
                 * @return {?}
                 */
                function (resource) {
                    _this.service.formFormio.userPermissions(_this.auth.user, form, resource).then((/**
                     * @param {?} perms
```

**Comment by jhen1422** [comment]
@dmenne 
Thank you for the report. We have created a ticket for the issue. We will review during our planning meeting next week. Please note, we welcome pull requests in this library. If you need this item expedited, please feel free to contact support@form.io for pricing.

**Comment by travist** [comment]
This should be fixed with latest 4.2.x releases.
